orchestrator/nodes/cron-batch/cron-batch.js

```diff
@@ -77,28 +77,32 @@ class DataHandler extends dojot.DataHandlerBase {
                 },
                 url: `http://cron:5000/cron/v1/jobs`,
                 data: JSON.stringify(jobRequest),
-                timeout: 1000
+                timeout: timeout
             }).then(response => {
                 let jobId = response.data.jobId;
                 logger.debug(`Succeeded to create cron job ${jobId}`, { filename: 'cron-batch' });
                 return resolve(jobId);
 
             }).catch(error => {
-                logger.error(`Failed to create cron job (${JSON.stringify(error.response.data)}).`, { filename: 'cron-batch' });
+                logger.error(`Failed to create cron job (${JSON.stringify(error)}).`, { filename: 'cron-batch' });
                 return reject(error);
             });
         });
     }
 
-    _createMultipleJobs(tenant, requests) {
+    _createMultipleJobs(tenant, requests, timeout) {
         let createPromises = [];
         for(let req of requests) {
-            createPromises.push(this._createSingleJob(tenant, req));
+            createPromises.push(this._createSingleJob(tenant, req, timeout));
         }
         return Promise.all(createPromises);
     }
 
     handleMessage(config, message, metadata) {
+        let timeout = config.timeout;
+        if (isNaN(timeout)) {
```

Comment:
Is this validation enough?  Can it be a negative number?

Reply:
good point, I'll fix it